docs/testing-contracts.rst

```diff
@@ -0,0 +1,241 @@
+.. index:: testing;testing;
+
+.. _testing:
+
+******************
+Testing a Contract
+******************
+
+This documentation recommends the use of the `pytest <https://docs.pytest.org/en/latest/contents.html>`_ framework with the `ethereum-tester <https://github.com/ethereum/ethereum-tester>`_ package.
+Prior to testing, the vyper specific contract conversion and the blockchain related fixtures need to be set up. These fixtures will be used in every test file and should therefore be defined in `conftest.py <https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions>`_.
+
+=================================
+Vyper Contract and Basic Fixtures
+=================================
+
+.. code-block:: python
+
+  import pytest
```

Comment:
Perhaps we should consider splitting our existing conftest.py into base_conftest.py and conftest.py which we can include here and it will stay up to date. In our existing conftest.py we just import the base functionality ;)

Reply:
Good idea. I'll make a proposition for this tomorrow and add it to the PR